tests/negative/security/tls.go

```diff
@@ -34,10 +34,18 @@ func init() {
 	config := &tls.Config{Certificates: []tls.Certificate{cer}}
 	customCAServer.TLS = config
 	customCAServer.Config.ErrorLog = log.New(ioutil.Discard, "", 0)
+
+	cer1, err := tls.X509KeyPair(publicKey1, privateKey1)
```

Comment:
Bikeshed: the use of `foo` and `foo1` throughout this commit seems confusing to me.  I'd expect `foo2` to come after `foo`.